source/_faq/release.markdown

```diff
@@ -1,12 +1,9 @@
 ---
 title: "Releases"
-description: "Releases"
+description: "Home Assistant release schedule"
 ha_category: Common
 ---
 
-The current release cycle is three weeks. This allows for two weeks of development time and 1 week of beta testing and fixing. Releases usually occur on a Wednesday. There is no fixed time of day when the release will happen because that depends on the person who is finishing the release, it could therefore be Tuesday or Thursday depending on your timezone when its available for you. If there was no announcement made in the previous release notes or on another communication channel then the release will happen.
+New major versions of Home Assistant are released every three weeks on a Wednesday. The exact dates can be seen in the upcoming events calendar on the [Home Assistant Developers](https://developers.home-assistant.io/) website. 
```

Comment:
Actually, minor, as we have not reached 1.0 yet.

Reply:
```suggestion
New versions of Home Assistant are released every three weeks on a Wednesday. The exact dates can be seen in the upcoming events calendar on the [Home Assistant Developers](https://developers.home-assistant.io/) website. 
```
My intention with "major" was to try and differentiate between feature releases and hotfixes. But in hindsight that doesn't belong here. I'll remove the word completely.